index.js

```diff
@@ -2,8 +2,8 @@
 const execa = require('execa');
 const del = require('del');
 const Listr = require('listr');
+const ansiEscapes = require('ansi-escapes');
 const split = require('split');
-require('any-observable/register/rxjs-all'); // eslint-disable-line import/no-unassigned-import
```

Comment:
Any reason why this is removed?

Reply:
Was going to use this commit to ask about that line but ended up not having time to write it yesterday. My bad.

After updating listr to version 0.13, I ran npm test and came across with this message: 
`/home/.../np/node_modules/any-observable/loader.js:18
			throw new Error('any-observable already defined as "' + registered.implementation +
			^
Error: any-observable already defined as "rxjs/Observable".  You can only register an implementation before the first  call to require('any-observable') and an implementation cannot be changed`

Thought that Observable declaration already made `any-observable/register/rxjs-all` a required dependency.